help/destinations/catalog/analytics/gainsight-px.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Gainsight PX 
+description: The Gainsight PX destination allows allows segmentation information to be pushed to Gainsight PX 
+---
+
+# Gainsight PX connection {#gainsight_px-destination}
+
+
+## Overview {#overview}
+
+[Gainsight PX](https://www.gainsight.com/product-experience/) is a product experience platform that enables product teams to understand how users use their products, collect feedback, and create in-app engagements like product walkthroughs to drive user onboarding and product adoption.
+
+>[!IMPORTANT]
+>
+>The destination connector and documentation page are created and maintained by the *Gainsight PX* team. For any inquiries or update requests, please contact them directly at *`pxsupport@gainsight.com`.*
+
+## Prerequisites {#prerequisites}
+
+* Contact Gainsight support to request the external segment features are enabled for your subscription
+* Generate an OAuth Secret value for your PX subscription, using the Generate New Secret button at the bottom of the Company Details page.
+
+## Supported identities {#supported-identities}
+
+
+*Gainsight PX* supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+| Target Identity |Description| Considerations                                                                                                                                                                                                                                                                         |
+|-----------------|---|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Please format the table with fewer dashes. 3 dashes are the minimum required for heading delimitation.